engines/scumm/imuse/imuse.cpp

```diff
@@ -1342,6 +1350,40 @@ void IMuseInternal::update_volumes() {
 	}
 }
 
+void IMuseInternal::musicVolumeReduction(MidiDriver *midi) {
+	int curVol;
+	int curEffVol;
+	int factor = 2; // The music volume variables are 0-255, and we need 0-127
+
+	if (_paused)
+		return;
+
+	_musicVolumeReductionTimer += midi->getBaseTempo();
+	while (_musicVolumeReductionTimer >= 16667) {
+		_musicVolumeReductionTimer -= 16667;
+		curVol = _music_volume / factor;
+
+		if (_vm->_sound->speechIsPlaying())
+			curVol = (90 * curVol) >> 7;
```

Comment:
Why 90? Perhaps a define or a comment would explain it better?

Reply:
Disasm :-D